rclcpp/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<package format="3">
```

Comment:
If we skip the installation of generated messages, we can revert this change and remove the `<member_of_group>` tag below.